src/test/java/org/tarantool/jdbc/JdbcStatementIT.java

```diff
@@ -125,15 +181,42 @@ public void testSupportedGeneratedKeys() throws SQLException {
     }
 
     @Test
-    void testUnsupportedGeneratedKeys() {
+    public void testExecuteNoGeneratedKeys() throws SQLException {
+        boolean isResultSet = stmt.execute(
+            "INSERT INTO test(id, val) VALUES (60, 'sixty')",
+            Statement.NO_GENERATED_KEYS
+        );
+        assertFalse(isResultSet);
+        ResultSet generatedKeys = stmt.getGeneratedKeys();
+        assertNotNull(generatedKeys);
```

Comment:
Is it worth to check whether the result set of generated keys is empty?

Reply:
Just check whether we follow the [API](https://docs.oracle.com/javase/8/docs/api/java/sql/Statement.html#getGeneratedKeys--):
> If this Statement object did not generate any keys, an empty ResultSet object is returned.

Reply:
I mean whether we must return the empty result set when NO_GENERATED_KEYS is set? If so it worth to check that the result set is empty.

Reply:
Yes, we must. 
Added one more assertion to check that no results are available.